src/webapp/client/index.html

```diff
@@ -213,23 +215,56 @@ <h2><center>Course Management</center></h2>
 				input fields followed by a button called "add course" which would
 				send the infomation to the webserver before it is sent to the
 				database.
+
+				All of these options are inclosed in a drop down box so that the user
+				can select to view these options or not to save screen space.
 				-->
-				<div class="row">
-					<div class="input-field col s12 m6 l3">
-						<input placeholder="CS/MAT165" id="courseName">
-						<label for="courseName">Course Name</label>
-					</div>
-					<div class="input-field col s12 m6 l3">
-						<input placeholder="Discrete Mathmatics" id="courseTitle">
-						<label for="couseTitle">Course Title</label>
-					</div>
-					<div class="input-field col s12 m6 l3">
-						<input placeholder="4" id="courseCredits">
-						<label for="couseCredits">Number of Credits</label>
-					</div>
+				<ul id="addCourseBox" class="collapsible" data-collapsible="expandable">
+					<li>
+						<div class="collapsible-header">
+							<i id="dbInfoArrow" class="material-icons">keyboard_arrow_down</i>Add A Course
+						</div>
+						<div class="collapsible-body">
+							<div class="row">
+								<div class="input-field col s12 m6 l3">
+									<input placeholder="CS/MAT165" id="courseName">
+									<label for="courseName">Course Name</label>
+								</div>
+								<div class="input-field col s12 m6 l3">
+									<input placeholder="Discrete Mathmatics" id="courseTitle">
+									<label for="couseTitle">Course Title</label>
+								</div>
+								<div class="input-field col s12 m6 l3">
+									<input placeholder="4" id="courseCredits">
+									<label for="couseCredits">Number of Credits</label>
+								</div>
+
+								<a id="btnAddCourse" class="waves-effect waves-light btn">Add Course</a>
+							</div><!--End of add course row.-->
+						</div>
+					</li>
+				</ul><!--End of add course drop down menu-->
+
+				<ul id="removecourseBox" class="collapsible" data-collapsible="expandable">
+					<li>
+						<div class="collapsible-header">
+							<i id="dbInfoArrow" class="material-icons">keyboard_arrow_down</i>Remove A Course
+						</div>
+						<div class="collapsible-body">
+							<div class="row">
+								<div class="input-field col s12 m6 l3">
+									<input id="courseName">
```

Comment:
So, i dont know a ton about this, but I think if you have more than one element with the same id, for example you have two input boxes named courseName, here and above, when you click the button to send to the server I think it'll send the first element it finds. 

Meaning if someone enters something in the addCourse section then again in the removeCourse section it will send the data thats in the addCourse section when you want to send the data in the removeCourse section.

Least I think thats how it works, I could be wrong though.